consensus/polybft/consensus_runtime.go

```diff
@@ -957,7 +959,10 @@ func (c *consensusRuntime) getSystemState(header *types.Header) (SystemState, er
 		return nil, err
 	}
 
-	return c.config.blockchain.GetSystemState(c.config.PolyBFTConfig, provider), nil
+	return c.config.blockchain.GetSystemState(
+		contracts.ValidatorSetContract,
```

Comment:
Though this is correct since these contracts are hardcoded, changing this is out of the scope of the decoupling of the rootchain contracts.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f35ba13ffe9fa37fd11178fe49e55f2f7c6f73d4